types/src/lib/config/auth.d.ts

```diff
@@ -0,0 +1,13 @@
+export declare type AUTH_USER_ROLE = 'Topcoder User' | 'Topcoder Member' | 'Topcoder Customer';
```

Comment:
The strings "Topcoder User", "Topcoder Member" and "Topcoder Customer" are used in at least two places. Can we encapsulate these as constants in one place and refer to the constants elsewhere? That way, if we ever need to change the actual string that's used to define the roles, we only need to update them in one place.

Reply:
Sure, I'll see if it can be improved. As for the `types` folder goes, these are automatically generated on build, they will still keep a text copy, and will not depend on the source files.